### PR TITLE
Add use_bias kwarg to LSTMCell and GRUCell

### PR DESCRIPTION
**Pull request by lungd**

### CODE REVIEW

**Comment by lungd** [comment]
Waiting for approval

File: src/layers/recurrent.jl [hunk 1]
```diff
@@ -211,21 +220,36 @@ function (lstm::LSTMCell)(x::AbstractMatrix, ps::Union{ComponentArray, NamedTupl
...
 end
 
-function (lstm::LSTMCell)((x, hidden_state,
-                           memory)::Tuple{<:AbstractMatrix, <:AbstractMatrix,
-                                          <:AbstractMatrix},
-                          ps::Union{ComponentArray, NamedTuple}, st::NamedTuple)
+function (lstm::LSTMCell{true})((x, hidden_state,
+                                 memory)::Tuple{<:AbstractMatrix, <:AbstractMatrix,
+                                                <:AbstractMatrix},
+                                ps::Union{ComponentArray, NamedTuple}, st::NamedTuple)
```
Comment (avik-pal) [inline src/layers/recurrent.jl:226]: We can remove `::Union{ComponentArray, ...}` now from all the calls.

**Review by avik-pal — CHANGES_REQUESTED** [review]
Also let's get a version bump

File: src/layers/recurrent.jl (hunk 1 above)
Comment (lungd) [reply src/layers/recurrent.jl]: Done

**Review by avik-pal — APPROVED** [review]

**Pull request merged**